### PR TITLE
chore(experimental,focal image): introducing focal statement as per support case

### PR DESCRIPTION
**Pull request by igpetrov**
Ref: https://github.com/camunda/connectors-bundle/issues/442

### CODE REVIEW

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        uses: docker/login-action@v2
+        with:
+          username: ${{ steps.secrets.outputs.DOCKERHUB_USER }}
+          password: ${{ steps.secrets.outputs.DOCKERHUB_PASSWORD }}
+          
+      - name: [FOCAL] Build and Push Docker Image tag ${{ github.event.inputs.version }} - bundle-default
+        uses: docker/build-push-action@v4
+        with:
+          context: bundle/mvn/default-bundle/
+          push: ${{ github.ref == 'refs/heads/main' }}
```
Comment (chillleader) [inline .github/workflows/RELEASE-FOCAL.yaml:85]: I find it useful to test such things in GHA before merging by running the workflow from the branch - you can do this by commenting/removing this line 🙂

**Review by chillleader — APPROVED** [review]
LGTM, but it might make sense to also test this by running the workflow from branch (see comment)

File: .github/workflows/RELEASE-FOCAL.yaml (hunk 1 above)
Comment (igpetrov) [reply .github/workflows/RELEASE-FOCAL.yaml:85]: Good point!

Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:85]:
```suggestion
          push: true
```

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          platforms: linux/amd64,linux/arm64
+          provenance: false
+          file: Dockerfile-focal
+          
+      - name: [FOCAL] Build and Push Docker Image tag latest - bundle-default
+        if: ${{ steps.check_prerelease.outputs.isPreRelease == 'false' }}
+        uses: docker/build-push-action@v4
+        with:
+          context: bundle/mvn/default-bundle/
+          push: ${{ github.ref == 'refs/heads/main' }}
```
Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:96]:
```suggestion
          push: true
```

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
+name: [FOCAL] Release a new version
```
Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:1]:
```suggestion
name: Focal: Release a new version
```

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+run-name: [FOCAL] Release a new version ${{ inputs.version }}
```
Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:2]:
```suggestion
run-name: Focal: Release a new version ${{ inputs.version }}
```

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        uses: docker/build-push-action@v4
+        with:
+          context: bundle/mvn/default-bundle/
+          push: true
+          tags: camunda/connectors-bundle:${{ github.event.inputs.version }}-focal
+          platforms: linux/amd64,linux/arm64
+          provenance: false
+          file: Dockerfile-focal
+          
+      - name: [FOCAL] Build and Push Docker Image tag latest - bundle-default
```
Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:91]:
```suggestion
      - name: Focal: Build and Push Docker Image tag latest - bundle-default
```

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 6]
```diff
@@ -0,0 +1,100 @@
+name: Focal: Release a new version
```
Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:1]:
```suggestion
name: Focal Release a new version
```

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+run-name: Focal: Release a new version ${{ inputs.version }}
```
Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:2]:
```suggestion
run-name: Focal Release a new version ${{ inputs.version }}
```

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+      - name: Set up Docker Build
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ steps.secrets.outputs.DOCKERHUB_USER }}
+          password: ${{ steps.secrets.outputs.DOCKERHUB_PASSWORD }}
+          
+      - name: [FOCAL] Build and Push Docker Image tag ${{ github.event.inputs.version }} - bundle-default
```
Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:81]:
```suggestion
      - name: Focal Build and Push Docker Image tag ${{ github.event.inputs.version }} - bundle-default
```

File: .github/workflows/RELEASE-FOCAL.yaml [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+        uses: docker/build-push-action@v4
+        with:
+          context: bundle/mvn/default-bundle/
+          push: true
+          tags: camunda/connectors-bundle:${{ github.event.inputs.version }}-focal
+          platforms: linux/amd64,linux/arm64
+          provenance: false
+          file: Dockerfile-focal
+          
+      - name: Focal: Build and Push Docker Image tag latest - bundle-default
```
Comment (igpetrov) [inline .github/workflows/RELEASE-FOCAL.yaml:91]:
```suggestion
      - name: Focal Build and Push Docker Image tag latest - bundle-default
```

**Comment by chillleader** [comment]
BTW testing the build workflows from the bundle repository can be painful since it takes a while to build. If you want to just test the docker build workflow you can use https://github.com/camunda/connector-runtime-docker as a playground where the builds finish like 10x faster :slightly_smiling_face:

**Pull request merged**